tests/test_gpcca.py

```diff
@@ -524,6 +525,7 @@ def test_gpcca_brandts_sparse_is_not_densified(self, P: np.ndarray, sd: np.ndarr
         with pytest.raises(ValueError, match=r"Sparse implementation is only available for `method='krylov'`."):
             GPCCA(csr_matrix(P), eta=sd, method="brandts").optimize(3)
 
+    @pytest.mark.skipif(Version(np.__version__) >= Version("2"), reason="Eigenvalue mismatch.")
```

Comment:
So you skip this test if numpy version is >=2? Why is it ok for newer numpy versions to fail this test?

Reply:
I've been trying to figure out why new `numpy` version causes this test to fail, but was unable to figure it out. I will try later do a bisect on the commits, see exactly why, but was thinking in the meantime we could just skip it.

Reply:
I've reverted this, will leave this for a future PR to investigate.